src/main/java/dk/cngroup/lentils/service/PdfExportService.java

```diff
@@ -0,0 +1,102 @@
+package dk.cngroup.lentils.service;
+
+import com.itextpdf.text.BaseColor;
+import com.itextpdf.text.Document;
+import com.itextpdf.text.DocumentException;
+import com.itextpdf.text.Element;
+import com.itextpdf.text.Font;
+import com.itextpdf.text.FontFactory;
+import com.itextpdf.text.Paragraph;
+import com.itextpdf.text.Phrase;
+import com.itextpdf.text.pdf.PdfPCell;
+import com.itextpdf.text.pdf.PdfPTable;
+import com.itextpdf.text.pdf.PdfWriter;
+import dk.cngroup.lentils.entity.view.TeamScore;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.util.List;
+import java.util.stream.Stream;
+
+@Service
+public class PdfExportService {
+    private static final int NUMBER_OF_TABLE_COLUMNS = 3;
+    private static final int PARAGRAPH_FONT_SIZE = 35;
+    private static final int SPACING_AFTER_PARAGRAPH = 20;
+
+    private final ScoreService scoreService;
+
+    @Autowired
+    public PdfExportService(final ScoreService scoreService) {
+        this.scoreService = scoreService;
+    }
+
+    public ByteArrayInputStream exportScoresToPdf() {
+        Document document = new Document();
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
+
+        try {
+            PdfWriter.getInstance(document, out);
+        } catch (DocumentException e) {
+            e.printStackTrace();
```

Comment:
Pokud nastane problem, chtelo by to dat vedet uzivateli (tj. poslat nejakou odpoved), ze se neco pokazilo.